website/pages/docs/k8s/ambassador.mdx

```diff
@@ -285,8 +285,8 @@ If you have tried the above troubleshooting steps and are still stuck, DataWire
 [ambassador]: https://www.getambassador.io/
 [ingress controller]: https://blog.getambassador.io/kubernetes-ingress-nodeport-load-balancers-and-ingress-controllers-6e29f1c44f2d
 [proxies]: /docs/connect/proxies
-[service sync]: /docs/platform/k8s/service-sync
-[connect sidecar]: /docs/platform/k8s/connect
+[service sync]: /docs/k8s/service-sync
+[connect sidecar]: /docs/k8s/connect
```

Comment:
You;ll need to add these to the [redirects list](https://github.com/hashicorp/consul/blob/master/website/_redirects)

Reply:
It's there already: https://github.com/hashicorp/consul/blob/master/website/_redirects#L184